.github/workflows/test.yml

```
@@ -80,7 +84,7 @@ jobs:
     needs:
       - ansible-lint
       - build-hugo-test
-    if: needs.check.outputs.ansible_changed == 'false' && needs.check.outputs.github_changed == 'false'
+    if: needs.check.outputs.ansible_changed != 'true' && needs.check.outputs.github_changed != 'true'
```

```suggestion
    if: needs.check.outputs.ansible_changed != true && needs.check.outputs.github_changed != true
```

Irgendwie funktioniert die "Sicherung gegen das Bearbeiten von falschen Dateien" noch nicht. Ich würde gern verhindern, dass Veränderungen an den ansible- oder github actions Dateien auf ein "Test"-Deployment gemacht werden können. Test und Prod laufen ja aktuell auf demselben Server.

---

Das ist für diesen PR nervig, weil wir ja die Oberfläche anpassen wollen. Sonst kann ich diese Funktionalität in einen anderen PR überführen.

---

Ach das habe ich gar nicht gesehen. Hab einfach auf dem Branch weitergearbeitet, die Änderung ist noch von dir oder?

---

Hab die Änderungen von mir raus genommen 👍

---

@juliaen Die Änderungen aus https://github.com/Feuerwehr-Kronshagen/homepage/pull/93 sind im main-branch. Vielleicht magst du einen rebase machen und schauen, ob für diesen PR ein Deployment ausgeführt wird? Danke Dir 🍀

---

Habs getestet und einen neuen Fix geschrieben. Ist im main 🎉